src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -19,49 +19,22 @@
   - along with this program. If not, see <http://www.gnu.org/licenses/>.
   -
   -->
+<docs>
+###	Example usage
+</docs>
+
 <template>
-	<!-- Navigation item -->
-	<nav-element :id="item.id" v-bind="navElement(item)"
-		:title="item.title" :class="[{'icon-loading-small': item.loading, 'open': opened, 'collapsible': collapsible }, item.classes]">
+	<!-- Navigation item, can be either an <li> or a <router-link> depending on the props -->
+	<nav-element v-bind="navElement()"
+		:title="title"
+		:class="[{'icon-loading-small': loading, 'open': opened, 'collapsible': collapsible }]"
+		@click.prevent.stop="">
 		<!-- Bullet -->
-		<div v-if="item.bullet" :style="{ backgroundColor: item.bullet }" class="app-navigation-entry-bullet" />
+		<div v-if="bullet !== ''" :style="{ backgroundColor: bullet }" class="app-navigation-entry-bullet" />
```

Comment:
It's better to have computeds for that, much cleaner
`haveBullet`

Reply:
`v-if="bullet"` should be fine, too, I think.